### PR TITLE
Edits to OIE Tuscany embedded app use cases - part 2

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Edits to OIE Tuscany embedded app use cases - part 2 (Pwd recovery)
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-413125](https://oktainc.atlassian.net/browse/OKTA-413125)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md [hunk 1]
```diff
@@ -147,5 +133,4 @@ switch (authnResponse.AuthenticationStatus)
 ...
 ```
 
-If the response's `AuthenticationStatus` returns `Success`, the change password flow has
-completed successfully and you can redirect the user to the default page.
+If the call to `ChangePasswordAsync()` returns a `AuthenticationStatus.Success` status, then the change password flow completed successfully and the user can be redirected to the default page.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md:136]:
```suggestion
If the call to `ChangePasswordAsync()` returns an `AuthenticationStatus.Success` status, then the change password flow completed successfully and the user can be redirected to the default page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/index.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ Before you build a password recovery flow with an email factor, ensure that your
...
 ### Set email as the only factor enabled for password recovery
 
-In addition to configuring your Okta org for the multifactor use case, you need to enable email as the only factor for password recovery.
+In addition to configuring your Okta org for the multifactor use case, you need to enable email as the only factor for password recovery for this flow.
 
 1. In the Admin Console, go to **Security** > **Authenticators**.
-1. On the **Authenticators** page, click **Actions** and then **Edit** for the **Password** authenticator.
-1. In the **Password** page, scroll down to the bottom of the page for the **Default Policy** and click
+1. From the **Setup** tab, select **Edit** from the **Actions** drop-down menu on the **Password** authenticator row.
+1. In the **Password** page, scroll down to the **Add Rule** section of the **Default Policy** and click
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/index.md:32]: it's "on pages" and "in dialog boxes"
```suggestion
1. On the **Password** page, scroll down to the **Add Rule** section of the **Default Policy** and click
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md [hunk 3]
```diff
@@ -110,26 +100,22 @@ switch (authnResponse.AuthenticationStatus)
...
+![Displays an example change password page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-change-pwd.png)
 
 </div>
 
 ### 9: Make a call to the ChangePasswordAsync method
 
-The final step is to make a call to `ChangePasswordAsync` to change the
-user's password. Pass `ChangePasswordOptions` into the method call with
-the `NewPassword` property set to the new password.
+The final step is to make a call to the `ChangePasswordAsync()` method to change the
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md:117]: weird that the sentence breaks right here....not a big deal...

**Review by susanharper-okta — APPROVED** [review]
really minor nits - approving so not to hold you up

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md:117]: I'll combine the three separate lines for the sentence into one.

**Pull request merged**